### DISCUSSION TITLE
Add examples to documentation in custom functions.

### DISCUSSION THREAD

**Discussion by siph** [nushell/nushell#8432] [Q&A]
The [book](https://www.nushell.sh/book/custom_commands.html#documenting-your-command) covers documenting functions and parameters but I can't figure out if it's possible to add examples like some built-in functions have.

eg: `split words --help`

**Comment by fdncred** [comment]
Agreed. It would be great to have `# Examples` or some such designation where we could document examples in our custom commands.
